cert-manager/overlays/moc/infra/api-cert.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: api-certificate-letsencrypt
+  namespace: openshift-config
```

Comment:
@HumairAK will we need a separate issuer in `openshift-config` namespace for this cert?